tests/unit/test_cli.py

```diff
@@ -131,7 +131,12 @@ def test_run_init_flask(mocker, tmp_path, monkeypatch):
 
     assert len(rock_project_yaml["summary"]) < 80
     assert len(rock_project_yaml["description"].split()) < 100
-    assert mock_ended_ok.mock_calls == [call()]
-
+    emitter.assert_message(
+        textwrap.dedent(
+            """\
+        Created 'rockcraft.yaml'.
```

Comment:
do you specifically want the single quotes around rockcraft.yaml?

Reply:
I did say I am following `snapcraft init`, which produces:

    Created 'snap/snapcraft.yaml'.
    Go to https://docs.snapcraft.io/the-snapcraft-format/8337 for more information about the snapcraft.yaml format.